src/components/CollaboratorModal/submodals/RemoveCollaboratorSubmodal.tsx

```diff
@@ -51,25 +66,17 @@ const RemoveCollaboratorSubmodalContent = () => {
       )}
 
       <Stack spacing={4} direction="row" justify="right" mt="36px">
-        <Button
-          variant="clear"
-          color="secondary"
-          onClick={() => {
-            setModalState(CollaboratorModalState.Default)
-            setDeleteCollaboratorTarget({})
-            closeModal()
-          }}
-        >
+        <Button variant="clear" color="secondary" onClick={onClose}>
           Cancel
         </Button>
         <Button
           colorScheme="danger"
-          onClick={() => {
+          onClick={async () => {
             if (userIsDeletingThemselves) {
               setRedirectToPage(`http://localhost:3000/sites`)
```

Comment:
nit: oof can we use `RouterLink` instead rather than hard coding the page link out? (trivial comment, feel free to dismiss this)

Reply:
`RouterLink` is meant to be used in a declarative way - if this was `RouterLink`, we would have to either always redirect them to `/sites` or to `""` @__@ i think this is a lil weird (we should probably have a context doing this auto-magically) but it's ok for now